libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -401,12 +402,29 @@ public MapboxMap getMapboxMap() {
     return map;
   }
 
+  public void setInstructionViewCallback() {
+    if (instructionView != null) {
```

Comment:
Would the `instructionView` ever be `null` here?  It's being bound right before this in `NavigationView#bind()`